bom/pom.xml

```diff
@@ -823,6 +823,11 @@
                 <artifactId>helidon-integrations-oci-metrics-cdi</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <dependency>
```

Comment:
@barchetta fyi - please review/approve poms and deps.

Reply:
You're only adding a bom entry, the pom.xml changes have nothing sensitive that requires @barchetta to approve.